pages/restaurants/reserve/[id].js

```diff
@@ -6,7 +6,7 @@ import ReserveForm from "../../../components/ReserveForm/ReserveForm.js";
 import RemainingSeatsFilter from "../../../components/RemainingSeatsFilter/RemainingSeatsFilter";
 import { useState } from "react";
 
-export default function Reseve({ handleReserve }) {
+export default function Reseve({ onReserve }) {
```

Comment:
I guess you have a typo here: Did you want to call the component `Reserve`?

Reply:
This is a function. it stores the data of reserve in local data after submit. I saw review conversation from a colleague, the function as props should be named with "on".  😝 So I changed the name as onReserve. I am really not good at entitling. Could you please give me a suggestion?